File: app/src/main/java/com/lifedawn/bestweather/retrofit/responses/metnorway/locationforecast/timeseries/Next6Hours.java
package com.lifedawn.bestweather.retrofit.responses.metnorway.locationforecast.timeseries;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Next6Hours {
	@Expose
	@SerializedName("summary")
	private Summary summary;
	
	@Expose
	@SerializedName("details")
	private Details details;
}

File: app/src/main/java/com/lifedawn/bestweather/data/remote/retrofit/responses/metnorway/locationforecast/timeseries/Next6Hours.kt
package com.lifedawn.bestweather.data.remote.retrofit.responses.metnorway.locationforecast.timeseries

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Next6Hours {
    @Expose @SerializedName("summary") var summary: Summary? = null
    @Expose @SerializedName("details") var details: Details? = null
}